src/chapters/inheritance/testing-inheritance.rst

```diff
@@ -0,0 +1,92 @@
+Testing Inheritance
+===================
+
+Not sure you get the whole inheritance idea? Still not sure which fields and methods get inherited and 
+which are overridden? Looking to *test* your understanding? 😉
```

Comment:
This emoji is not working very well with a screen reader. I would recommend removing it.